src/area-select.lite.tsx

```diff
@@ -14,6 +14,12 @@ export const AreaSelect = ({
 }: AreaSelectProps) => {
   const { areas } = useContext(configContext);
 
+  const filterSort: SelectProps['filterSort'] = (a, b) => {
```

Comment:
I think we can just do the sorting in provider and remove the `filterSort` usage.

Reply:
That's true. Let's simplify it.